packages/client/subscriptions/TeamSubscription.ts

```diff
@@ -40,6 +40,7 @@ const subscription = graphql`
   subscription TeamSubscription {
     teamSubscription {
       __typename
+      ...SetDefaultSlackChannelMutation_team @relay(mask: false)
```

Comment:
remember, if you've published to a subscription channel, you have to include the updater here
(the newMutation CLI script does this for you automatically if you use it 😉 )

Reply:
The newMutation CLI is super helpful - I used it for this mutation but it doesn't look like it adds an updater function by default. I could take a crack at adding it to the script if you like?